### PR TITLE
Initial commit for testing

### PR DESCRIPTION
**Pull request by ameyapat**
## Proposed changes

Test PR to test branch restrictions on release pipeline

## Type of change

- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: azure_pipelines/changes.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+1
```
Comment (kaisong1990) [inline azure_pipelines/changes.txt:1]: This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.

File: azure_pipelines/pipeline-trigger.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    value: 'AzureAD/microsoft-authentication-library-for-objc'
+  - name: 'repositoryBranch' # Name of the branch where version number will be updated for files that contain version number.
+    value: 'ameya-master'
+  - name: 'GithubServiceConnection' # Service connection name used to connect Github repository
+    value: 'MSAL ObjC Service Connection'
+
+trigger:
+  branches:
+    include:
+    - ameya-master
```
Comment (kaisong1990) [inline azure_pipelines/pipeline-trigger.yml:18]: Will this be master branch eventually?

**Review by kaisong1990 — APPROVED** [review]
Approved with a minor question

File: azure_pipelines/pipeline-trigger.yml (hunk 2 above)
Comment (ameyapat) [reply azure_pipelines/pipeline-trigger.yml:18]: No, this is just for testing. I'll both branches when my testing is done

**Pull request merged**